jscomp/main/bsb_main.ml

```diff
@@ -131,6 +134,23 @@ let handle_anonymous_arg arg =
 let program_exit () =
   exit 0
 
+let install_target config_opt =
+  let config =
+    match config_opt with
+    | None ->
+    let config = Bsb_config_parse.interpret_json ~toplevel_package_specs:None ~per_proj_dir:Bsb_global_paths.cwd in
+    let _ = Ext_list.iter config.file_groups.files (fun group -> 
+    let check_file = match group.public with
```

Comment:
For the record, this has some duplication with Bsb_ninja_file_groups.handle_files_per_dir, I will handle it later